jest.config.js

```diff
@@ -39,8 +40,13 @@ module.exports = {
   setupFiles: ['<rootDir>/config-jest/setup/shims.js', '<rootDir>/config-jest/setup/console.js'],
   setupFilesAfterEnv: ['<rootDir>/config-jest/setup/enzyme.js'],
   testEnvironment: 'jsdom',
-  testURL: 'http://localhost',
-  timers: 'fake',
+  testEnvironmentOptions: {
+    url: 'http://localhost',
+  },
+  fakeTimers: {
+    enableGlobally: true,
+    legacyFakeTimers: true,
```

Comment:
There is only a single test that fails when using the modern mock timers, but it wasn't immediately obvious how to fix this (probably just `jest.useRealTimers()`?), so I've just kept using the legacy fake timers for now which were the default with jest 25.

https://github.com/airbnb/visx/blob/0614604ece30ac9e14b58853dbc8b05e86a39627/packages/visx-mock-data/test/genDateValue.test.ts#L21

failed with 

```raw
 FAIL  packages/visx-mock-data/test/genDateValue.test.ts
  ● generators/genDateValue › should return [{ date, value }]

    expect(received).toEqual(expected) // deep equality

    Expected: [Function ClockDate]
    Received: [Function Date]

      19 |     const n = 1;
      20 |     const data = genDateValue(n);
    > 21 |     expect(data[0].date.constructor).toEqual(Date);
         |                                      ^
      22 |     expect(typeof data[0].value).toBe('number');
      23 |   });
      24 |

      at Object.toEqual (packages/visx-mock-data/test/genDateValue.test.ts:21:38)
```
---
There is another test referencing fake timers, so that one should be revisited as well when switching from legacy to modern fake timers:

https://github.com/airbnb/visx/blob/0614604ece30ac9e14b58853dbc8b05e86a39627/packages/visx-responsive/test/useScreenSize.test.ts#L36-L37